tools/testing/target_determination/heuristics/mentioned_in_pr.py

```diff
@@ -0,0 +1,83 @@
+import os
```

Comment:
Not sure why this new heuristic is here, it's worth a separate PR I think.

Reply:
oops probably rebased badly